cmd/virtual-kubelet/commands/root/root.go

```diff
@@ -147,12 +147,12 @@ func runRootCommand(ctx context.Context, c Opts) error {
 	}
 
 	pNode := NodeFromProvider(ctx, c.NodeName, taint, p)
-	node, err := vkubelet.NewNode(
-		vkubelet.NaiveNodeProvider{},
+	nodeRunner, err := node.NewNode(
```

Comment:
Suggestion:
- `node.NewNodeLifecycleManager`, or
- `node.NewNodeController` to follow similar semantics as `node.NewPodController`.

Reply:
I *almost* did this in the PR 😄

Reply:
Took your suggestion and change to NodeController

Reply:
This does cause golint to complain about stuttering.

Reply:
golint is too serious.